src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -8,7 +8,7 @@
 from ._client_factory import iot_service_provisioning_factory
 from ._client_factory import iot_central_service_factory
 
-JOB_DEPRECATION_INFO = 'IoT Extension (azure-cli-iot-ext) Job commands'
+CS_DEPRECATION_INFO = 'IoT Extension (azure-cli-iot-ext) connection-string commands'
```

Comment:
I think we should also show the equivalent command in the extension so the user doesn't have to look for it `az iot hub connection-string show`

Reply:
Done